applications/wika_network.md

```diff
@@ -0,0 +1,446 @@
+# W3F Open Grant Request
+
+
+* **Project Name:** Wika Network (temporary prototype name)
+* **Team Name:** Web Registry DAO
+* **Payment Address:** 0xBfc8cDb5617eE0F83794156ABD7465730E125cE3 (USDT)
+
+
+
+## Project Overview :page_facing_up:
+
+
+### Overview
+
+#### Make the Internet a Better Place.
+
+Our mission is to build a clean, public and non-intrusive blockchain registry to empower 
+authors, search engines and recommendation systems that respect privacy and are 100% transparent.
+
+- **Content Consumers** will be able to like their favorite pages and reward the authors and previous likers.
+
+- **Authors** will register their ownership in one public decentralized database, and whether their content lives in youtube, 
+facebook, medium or any other internet place, Wika users will be able to reward them directly.
+ 
+#### Why?
+
+Because there are too many Like buttons out there!
+
+Each one generates data in private databases that are often used to optimize for advertisment or hidden agendas.
```

Comment:
The issue I see here is that, instead of allowing one company to process my likes, I now allow everyone to process them. That's not exactly an improvement. And wouldn't I have to give a website that integrates your like button access to the accounts in my wallet? That would allow them to link any other data they have of me with all the likes in your registry.

Reply:
Very good point!
When a website integrates the plugin, it doesn't solve the data privacy problem as you point out, it even gives them access to more data, but one could argue that it's still a step towards progress.
Why? We think that the direct integration in websites is an intermediate step towards web3 to gain more mass adoption.

First of all, in that setting, the user doesn't have to be logged in anymore, or can use an anonymous account.
Secondly, we expect the widget to be first integrated by actors who actually want to promote web3 rather than very powerful web2 companies, and by doing so, will give us a user base ready to upgrade to the browser extension version, or even polkadot wallets that provide wika functionality in a seamless and protected fashion.
The ideal setting is when the browser is equipped with an extension that hides the actual address from the websites and only injects the widget and the like functionality. In that case the website will only be able to record the like action without an actual handle on the address.
 
My point is that by providing a web3 compliant infrastructure for registering web pages and liking, the world should progressively evolve into using it in the ideal way.
Somehow, this question applies to web3 in general: as I've been connecting my Metamask and Polkadot extension to so many websites already, it's true that I didn't share my personal credentials, but I also shared much more data. I also added my email address to opensea to receive offers, so now not only opensea has my personal info, but it also has all my wallet activity all the way to Uniswap and Aave. Not ideal from a data privacy standpoint at all, but i still believe that using opensea is a positive step towards web3 evolution.

This is actually the first time I think about the problem the way you pose it and this is just my intuitive immediate reaction. I will share this with the rest of the team and discuss further about what other ways we could come up to address it. And also please help me with any useful links on the subject or ways you think the infrastructure could be better designed. We are definitely here to learn!